src/resource/nonRdfData.ts

```diff
@@ -79,9 +81,10 @@ export async function getFile(
   };
   const url = internal_toIriString(input);
   const response = await config.fetch(url, config.init);
-  if (!response.ok) {
-    throw new Error(
-      `Fetching the File failed: ${response.status} ${response.statusText}.`
+  if (internal_isUnsuccessfulResponse(response)) {
+    throw new FetchError(
+      `Fetching the File failed: ${response.status} ${response.statusText}.`,
```

Comment:
Just to be consistent across all our code (not just all our JavaScript code, but our Java and Python code too), I'd delimit the variable placeholders:
```suggestion
      `Fetching the File failed: [${response.status}] [${response.statusText}].`,
```

Reply:
Actually I guess you'll suggest updating all occurrences of the backtick delimiter with `[` and `]` later in one specific PR, but personally, since this is brand new code, I'd just start using that convention now, and fix the rest of the code in that later PR (i.e. I wouldn't see a particularly strong need to maintain consistency within just this codebase), and it just means less updates to make in that subsequent clean-up PR.

Reply:
I did not touch the actual error message, but I added 63523897f74308b0e8a6968d353ef907f304a6b0 to add them now. I did use the delimiters we use elsewhere in the code.

Reply:
| I did use the delimiters we use elsewhere in the code.

But we've all agreed (including Jack, as tech lead for all front-end development at Inrupt) to use '[' and ']', and not backticks as delimiters, right? So why continue to use backticks on any new code we write (i.e. when someone is going to have to revert all of these to use '[' and ']')...?

Reply:
For consistency - we work through the whole codebase in one fell swoop in a future ticket to update the delimiters everywhere, rather than use one type in some places, and another type in other places.

Reply:
I'd prefer internal consistency, which we can upgrade all at once.

Reply:
Okey dokey - but I reckon my comments absolve me from being the lucky dev chosen to implement that 'switch to brackets everywhere' PR :) !
(BTW, I notice now that the GitHub 'Suggestions' button is greyed out now, and the helpful tooltip explains why (comment is outdated). But what does having no 'Suggestions' button at all mean (or is the greyed-out-and-tooltip thing a brand new update, like in the past 10 mins!)...?):
![image](https://user-images.githubusercontent.com/961445/98688010-4cf7af80-2362-11eb-9398-9a5aea4773b9.png)

Reply:
Haha sure, feel free to create the ticket and then mention those comments here :) 

I think adding a suggestion only works when commenting on a change that's still in the PR. (Which is really annoying if you want to comment on how something affects some other piece of code, but alas...)

Reply:
Sure, I had already created the ticket  (SDK-1391), but I've updated it to point here too :).